attributes/default.rb

```diff
@@ -493,18 +493,15 @@
 
 #DOCKER
 default['hops']['docker']['enabled']                  = "true"
-default['hops']['docker_version']['ubuntu']           = "19.03.6-0ubuntu1~18.04.2"
-default['hops']['docker_version']['centos']           = "19.03.8-3"
-default['hops']['selinux_version']['centos']          = "2.119.1-1.c57a6f9"
-default['hops']['containerd_version']['ubuntu']       = "1.2.6-0ubuntu1~18.04.2"
-default['hops']['containerd_version']['centos']       = "1.2.13-3.1"
-default['hops']['runc_version']['ubuntu']             = "1.0.0~rc95-0ubuntu1~18.04.1"
+default['hops']['docker_version']['centos']           = "20.10.9-3"
+default['hops']['docker_version']['ubuntu']           = "20.10.7"
 default['hops']['docker']['group_id']                 = '1513'
 default['hops']['docker']['storage_driver']           = "overlay2"
 
 
-default['hops']['docker']['pkg']['download_url']['centos'] ="#{node['download_url']}/docker/#{node['hops']['docker_version']['centos']}/rhel"
-default['hops']['docker']['pkg']['download_url']['ubuntu'] ="#{node['download_url']}/docker/ubuntu/#{node['hops']['docker_version']['ubuntu'].split('-')[0]}/"
+
+default['hops']['docker']['download_url']['centos'] = "https://repo.hops.works/dev/gibson/docker/20.10.9-3.tgz"
```

Comment:
Why did you change this? - it's available here and it works: https://repo.hops.works/master/docker/rhel/

Reply:
I updated containerd to the latest version 1.6.9. There was some issue with the older containerd you have, but I forgottent the error. I can try to reproduce it later.